posts/2022-04-12-java18-22004.adoc

```diff
@@ -253,7 +249,10 @@ The following 31 guides are updated to use MicroProfile 5 and Jakarta EE 9 featu
 * link:https://openliberty.io/guides/docker.html[Using Docker containers to develop microservices]
 * link:https://openliberty.io/guides/bean-validation.html[Validating constraints with microservices]
 
+[#paketo]
```

Comment:
On second thought, can you move this as the third item in the blog (as shown in the suggestion for the Toc), before the CVE section.